jetty9/src/main/docker/15-cdbg-env.bash

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+# Adds bits of configuration necessary for the Cloud Debugger to work with the Jetty image.
+# It needs to run before 20-debug-env.bash from the openjdk image.
+
+# Set CDBG_APP_WEB_INF_DIR, used by CDBG in format-env-appengine-vm.sh
+export CDBG_APP_WEB_INF_DIR="${JETTY_BASE}/webapps/root/WEB-INF"
```

Comment:
I would rename the file to `15-debug-env-jetty.bash` to protect a bit more against file collisions.